java/code/src/com/suse/manager/utils/PackageUtils.java

```diff
@@ -41,6 +43,40 @@ public static boolean isTypeDeb(Package pkg) {
         return "deb".equals(getArchTypeLabel(pkg));
     }
 
+    /**
+     * Get the channel architecture as a universal arch string, practically
```

Comment:
Given the definition of "universal" is new to this PR, I would spend a line describing it in comments.

Eg. "Universal means:..."

(I know this is implied by the code and existing comments - this is just to make life easier to future readers)

Reply:
Added a phrase in the descriptions